src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -460,24 +415,6 @@ void AppHost::Initialize()
     // set that content as well.
     _window->SetContent(_windowLogic.GetRoot());
     _window->OnAppInitialized();
-
-    // BODGY
-    //
-    // We've got a weird crash that happens terribly inconsistently, but pretty
-    // readily on migrie's laptop, only in Debug mode. Apparently, there's some
-    // weird ref-counting magic that goes on during teardown, and our
-    // Application doesn't get closed quite right, which can cause us to crash
-    // into the debugger. This of course, only happens on exit, and happens
-    // somewhere in the XamlHost.dll code.
-    //
-    // Crazily, if we _manually leak the Application_ here, then the crash
-    // doesn't happen. This doesn't matter, because we really want the
-    // Application to live for _the entire lifetime of the process_, so the only
-    // time when this object would actually need to get cleaned up is _during
-    // exit_. So we can safely leak this Application object, and have it just
-    // get cleaned up normally when our process exits.
-    auto a{ _app };

```

Comment:
oh no

Reply:
did anyone else _ever_ hit this?

Reply:
https://github.com/microsoft/terminal/pull/5629

Reply:
I mean, it definitely stopped teardown from happening... so I can't tell you how many crashes we _didn't_ get because of it!

Reply:
no worries, i can yeet that back in